src/models/extra_inputs.cpp

```diff
@@ -5,6 +5,46 @@
 
 namespace Generators {
 
+PresetExtraInputs::PresetExtraInputs(State& state)
+    : state_(state),
+      registry_{
```

Comment:
Is the registry always the same? Should this be a global that's created once vs per state?

Reply:
I think creating the registry is inexpensive and creating one over and over again per State should be ok for now.

I can make it global but would like to avoid making things global if possible.